docker/Dockerfile

```diff
@@ -1,6 +1,6 @@
 # docker build --rm -f docker/Dockerfile -t plugins/docker .
 
-FROM docker:17.10.0-ce-dind
+FROM docker:stable-dind
```

Comment:
We pin the plugin to a specific docker version and tag the plugin using the same version number as Docker. So we would accept `17.12.0-ce-dind` which is the same as `stable-dind` with a version bump in the `.drone.yml` for `17.12`

Reply:
@bradrydzewski thanks for the quick response, updated the PR.